tools/tuf/test/README.md

```diff
@@ -82,4 +82,20 @@ make desktop-app-tar-gz
 
 # Push the desktop target as a new version
 ./tools/tuf/test/push_target.sh macos desktop desktop.app.tar.gz 43
-```
\ No newline at end of file
+```
+
+### Troubleshooting
+
+#### Fleet Desktop Startup Issue on macOS
+
+When running Fleet Desktop on an older macOS version than it was compiled on, Orbit may not launch it due to an error:
+
+```
+_LSOpenURLsWithCompletionHandler() failed with error -10825
+```
+
+Solution: `Set the MACOSX_DEPLOYMENT_TARGET` environment variable to the lowest macOS version you intend to support:
```

Comment:
Set this when building Fleet Desktop?

Reply:
you mean we should clarify to set the env var only if/when you build Fleet Desktop?

Reply:
Yes, no big deal, is it a variable the `go` command picks up?

Reply:
I believe it's used by the macOS frameworks the systray library is linking. I could find [this](https://opensource.apple.com/source/cctools/cctools-667.10.0/libstuff/macosx_deployment_target.c.auto.html) and [this](https://developer.apple.com/library/archive/documentation/DeveloperTools/Reference/XcodeBuildSettingRef/1-Build_Setting_Reference/build_setting_ref.html#//apple_ref/doc/uid/TP40003931-CH3-SW129)

I will add a follow up PR with the suggested clarification, thanks!